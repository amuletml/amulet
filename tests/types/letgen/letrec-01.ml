let rec (a, b) = (a, b)
