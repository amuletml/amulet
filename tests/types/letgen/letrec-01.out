letrec-01.ml[1:5 ..1:10]: error
  Recursive pattern bindings are not allowed
  Note: this definition refers to itself directly

  │ 
1 │ let (a, b) = (a, b)
  │     ^^^^^^
